doc/20-HTTP-API.md

```diff
@@ -42,29 +42,29 @@ EOF
 
 ## Debugging Endpoints
 
-There are multiple endpoints for dumping specific configurations.
+There are multiple endpoints for dumping specific configurations. All of them are prefixed by `/debug`.
```

Comment:
At the moment, each sentence starts in its own line. This eases the readability of future diffs.

```suggestion
There are multiple endpoints for dumping specific configurations
All of them are prefixed by `/debug`.
```